src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexIsotopicPeakPattern.cpp

```diff
@@ -49,12 +49,12 @@ namespace OpenMS
     charge_(c), peaks_per_peptide_(ppp), mass_shifts_(ms), mass_shift_index_(msi)
   {
     // generate m/z shifts
-    for (unsigned i = 0; i < mass_shifts_.getMassShiftCount(); ++i)
+    for (unsigned i = 0; i < mass_shifts_.getDeltaMasses().size(); ++i)
     {
       for (int j = -1; j < peaks_per_peptide_; ++j)
       {
         // j=-1 shift corresponds to the zeroth peak
-        mz_shifts_.push_back((mass_shifts_.getMassShiftAt(i) + j * Constants::C13C12_MASSDIFF_U) / charge_);
+        mz_shifts_.push_back((mass_shifts_.getDeltaMasses()[i].delta_mass + j * Constants::C13C12_MASSDIFF_U) / charge_);
```

Comment:
this is fine but consider (especially in the future) that it is sometimes results in clearer code to use a const reference as in:
```
const vector<DeltaMass>& delta_masses = mass_shifts_.getDeltaMasses();
 mz_shifts_.push_back((delta_masses[i].delta_mass + j * Constants::C13C12_MASSDIFF_U) / charge_);
```

There should be no speed difference (it actually might be faster).